app/views/feedback/new.html.haml

```diff
@@ -0,0 +1,15 @@
+
+%section#content.content-main.form_wrapper{role: "main"}
+  %h1 Your Feedback
```

Comment:
The capitalisation style used on the rest of the site, is capital first word and subsequent words in lowercase (unless their proper nouns). We should probably do that for all the headings/labels in this form as well.